judge/runguard.cc

```diff
@@ -1175,7 +1178,7 @@ int main(int argc, char **argv)
 			preserve_environment = 1;
 			break;
 		case 'V': /* set environment variable */
-			environment_variables = strdup(optarg);
+			environment_variables.push_back(optarg);
```

Comment:
Nitpick: maybe make the implicit `string()` cast/constructor that creates a copy explicit?

Reply:
Done